resources/v8/02_books.sql

```diff
@@ -0,0 +1,7 @@
+
+ALTER TABLE books
+    ADD language_id int NOT NULL;
```

Comment:
i think you have to use different type to store array in this language.
and also if you are going to store multiple language rename it to
  just languages